strawberryfields/__init__.py

```diff
@@ -50,20 +51,18 @@
 -------------------
 
 .. autosummary::
-   Engine
    convert
    version
 
 Code details
 ~~~~~~~~~~~~
 """
 
-from __future__ import unicode_literals
 from ._version import __version__
-from .engine import Engine as _Engine
-from .engine import _convert as convert
+from .program import (Program, _convert as convert)
```

Comment:
I've forgotten what the reason is - why are we importing `_convert` as `convert`, and not just calling it `convert` to begin with?